### PR TITLE
[Exchanges] add BitMart tests

### PR DESCRIPTION
**Pull request by GuillaumeDSM**

### CODE REVIEW

File: tests_additional/real_exchanges/test_bitmart.py [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        tickers = await self.get_all_currencies_price_ticker()
+        for symbol, ticker in tickers.items():
+            self._check_ticker(ticker, symbol)
+
+    async def test_get_all_currencies_price_ticker_with_market_filter(self):    #todo
+        tickers = await self.get_all_currencies_price_ticker(market_filter=self.get_market_filter())
+        assert len(tickers) > 2    # all tickers
+        assert self.SYMBOL in tickers
+        assert self.SYMBOL_2 in tickers
+        assert self.SYMBOL_3 in tickers  # symbol not correctly parsed as not in available markets (but luckily kucoin also uses the same syntax)
```
Comment (Herklos) [inline tests_additional/real_exchanges/test_bitmart.py:145]: 😄

**Review by Herklos — APPROVED** [review]
🎉

File: tests_additional/real_exchanges/test_bitmart.py (hunk 1 above)
Comment (GuillaumeDSM) [reply tests_additional/real_exchanges/test_bitmart.py:145]: woops

**Comment by GuillaumeDSM** [comment]
up

**Pull request merged**